<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoleQueue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="errorNotificationLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;An error has occurred. Click to &lt;a href=&quot;viewLog&quot;&gt;view the log&lt;/a&gt; or &lt;a href=&quot;clearError&quot;&gt;remove this notification.&lt;/a&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MoleQueue::JobTableWidget" name="jobTableWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQueueManager"/>
    <addaction name="actionOpenWithManager"/>
    <addaction name="actionShowLog"/>
    <addaction name="actionUpdateRemoteQueues"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewJobFilter"/>
    <addaction name="actionAdvancedJobFilters"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>&amp;Jobs</string>
    </property>
    <addaction name="actionClearFinishedJobs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuView"/>
   <addaction name="menuJobs"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Mi&amp;nimize</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Ma&amp;ximize</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore</string>
   </property>
  </action>
  <action name="actionQueueManager">
   <property name="text">
    <string>Queue &amp;Manager</string>
   </property>
  </action>
  <action name="actionOpenWithManager">
   <property name="text">
    <string>&quot;Open With&quot; Manager...</string>
   </property>
   <property name="toolTip">
    <string>Configure &quot;Open with&quot; applications</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="text">
    <string>Show &amp;Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionUpdateRemoteQueues">
   <property name="text">
    <string>&amp;Update remote queues</string>
   </property>
   <property name="toolTip">
    <string>Update the status of jobs on remote queuing systems.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionViewJobFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Job &amp;Filter</string>
   </property>
   <property name="toolTip">
    <string>Show Job Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClearFinishedJobs">
   <property name="text">
    <string>Clear &amp;Finished Jobs</string>
   </property>
   <property name="toolTip">
    <string>Clear finished jobs from MoleQueue</string>
   </property>
  </action>
  <action name="actionAdvancedJobFilters">
   <property name="text">
    <string>Ad&amp;vanced Job Filters...</string>
   </property>
   <property name="toolTip">
    <string>Show advance job filtering options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoleQueue::JobTableWidget</class>
   <extends>QWidget</extends>
   <header>jobtablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
